test/nntrainer_test_util.cpp

```diff
@@ -48,3 +58,189 @@ void replaceString(const std::string &from, const std::string &to,
   data_file << s;
   data_file.close();
 }
+
+/**
+ * @brief     Generate Random integer value between min to max
+ * @param[in] min : minimum value
+ * @param[in] max : maximum value
+ * @retval    min < random value < max
+ */
+static int rangeRandom(int min, int max) {
+  int n = max - min + 1;
+  int remainder = RAND_MAX % n;
+  int x;
+  do {
+    x = rand();
+  } while (x >= RAND_MAX - remainder);
+  return min + x % n;
+}
+
+/**
+ * @brief     load data at specific position of file
+ * @param[in] F  ifstream (input file)
+ * @param[out] outVec
+ * @param[out] outLabel
+ * @param[in] id th data to get
+ * @retval true/false false : end of data
+ */
+static bool getData(std::ifstream &F, std::vector<float> &outVec,
+                    std::vector<float> &outLabel, int id) {
+  F.clear();
+  F.seekg(0, std::ios_base::end);
+  uint64_t file_length = F.tellg();
+  uint64_t position = (feature_size + num_class) * id * sizeof(float);
```

Comment:
I think, it would be nice to have handle the case when `id < 0`

Reply:
fixed. Thanks.